package.json

```diff
@@ -52,6 +52,15 @@
     "commitMessage": "Deploy Storybook [skip ci]"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.8",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@babel/plugin-proposal-decorators": "^7.12.1",
+    "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
+    "@babel/preset-env": "^7.12.7",
+    "@babel/preset-react": "^7.12.7",
+    "@babel/preset-typescript": "^7.12.7",
+    "@fluentui/web-components": "^0.22.1",
```

Comment:
This should be a dependency on the mgt-components package instead of the root package.json. Otherwise, the npm package and bundle won't pull in the dependency

Reply:
😳 Good to know